packages/stripe_ios/ios/Classes/Stripe Sdk/StripeSdk+PaymentSheet.swift

```diff
@@ -224,9 +224,11 @@ extension StripeSdk {
             mode = PaymentSheet.IntentConfiguration.Mode.payment(
                 amount: amount,
                 currency: modeParams["currencyCode"] as? String ?? "",
-                setupFutureUsage: modeParams["setupFutureUsage"] != nil
-                    ? (modeParams["setupFutureUsage"] as? String == "OffSession" ? .offSession : .onSession)
-                    : nil,
+                setupFutureUsage: { return switch modeParams["setupFutureUsage"] as? String {
```

Comment:
can you remove this change and let's go for a fix in dart or we need to make a change to stripe ios

Reply:
I thought to have removed it in last commit, while fixing it in dart, let me check when I return home

Reply:
But, a moment, this seems the rollback, compare it against  main branch

Reply:
Ok, check it now again, please